photon-core/src/main/java/org/photonvision/vision/frame/FrameStaticProperties.java

```diff
@@ -35,6 +37,9 @@ public class FrameStaticProperties {
     public final double verticalFocalLength;
     public CameraCalibrationCoefficients cameraCalibration;
 
+    private CameraCalibrationCoefficients[] cachedCalibrationCoefficients =
+            new CameraCalibrationCoefficients[4];
```

Comment:
IMO this cache belongs in CameraCalibrationCoefficients? Or can we cache the whole FrameStaticProperties from rotate? like how expensive is creating a new FrameStaticProperties every loop? doesn't seem fantastic for gc pressure.